include/bluetooth/direction.h

```diff
@@ -26,6 +26,87 @@ struct bt_df_adv_cte_tx_param {
 	uint8_t  *ant_ids;
 };
 
+enum BT_DF_CTE_RX_TYPE {
+	BT_DF_CTE_RX_TYPE_AOA = BIT(0),
+	BT_DF_CTE_RX_TYPE_AOD_1US = BIT(1),
+	BT_DF_CTE_RX_TYPE_AOD_2US = BIT(2),
+};
+
+#define BT_DF_CTE_RX_TYPE_ALL                                                  \
+	(BT_DF_CTE_RX_TYPE_AOA | BT_DF_CTE_RX_TYPE_AOD_1US |                   \
+	 BT_DF_CTE_RX_TYPE_AOD_2US)
+
+/* @brief Constant Tone Extension parameters for connectionless
+ * reception.
+ *
+ * @note cte_type is a bit field that provides information about type of CTE
+ * an application expects:
+ * - BT_DF_CTE_RX_TYPE_AOA for Angle of Arrival
+ * - BT_DF_CTE_RX_TYPE_AOD_1US for Angle of Departure with 1us switching slots
+ * - BT_DF_CTE_RX_TYPE_AOD_2US for Angle of Departure with 2us switching slots
+ *
+ * In case BT_DF_CTE_RX_TYPE_AOA is not set, members: slot_durations,
+ * num_ant_ids and ant_ids are not required and their values will be not verified
+ * for correctness.
+ */
+struct bt_df_per_adv_sync_cte_rx_param {
+	/* Bitmap with allowed CTE types: AoD 1us, AoD 2us, AoA */
+	uint8_t cte_type;
+	/** Antenna switching slots, one of possible values: 1us, 2us. */
+	uint8_t slot_durations;
+	/** Max number of CTEs to receive, 0 means receive continuously. */
+	uint8_t max_cte_count;
+	/** Length of antenna switch pattern */
+	uint8_t num_ant_ids;
+	/** Antenna switch pattern */
+	const uint8_t *ant_ids;
```

Comment:
What is the maximum number of antenna IDs?

Reply:
Maximum number of allowed antenna IDs is 75, That is a value provided by BT 5.1. spec. But in case of controller implementation for Nordic SOCs the value is limited to 39. Actual max value is read by hci_df_read_ant_info call to controller. Returned by the function max switch pattern length holds CONFIG_BT_CTLR_DF_MAX_ANT_SW_PATTERN_LEN value.